django/contrib/messages/storage/cookie.py

```diff
@@ -166,17 +168,23 @@ def _decode(self, data):
         if not data:
             return None
         try:
-            decoded = self.signer.unsign(data)
+            return self.signer.unsign_object(data, serializer=MessageSerializer)
+        # RemovedInDjango40Warning: when the deprecation ends, replace both with:
+        #
+        # except Exception:
+        #     pass
         except signing.BadSignature:
-            # RemovedInDjango40Warning: when the deprecation ends, replace
-            # with:
-            #   decoded = None.
```

Comment:
This solution is should be applied in Django 4.0. I don't see a reason to remove this comment.

Reply:
I might have mixed up 4.0and 4.1 to be honest.